<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BridgeDock</class>
 <widget class="QDockWidget" name="BridgeDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Entity Explorer Bridge</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="16" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0">
     <widget class="QLineEdit" name="lineEditLoadFolder">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QPushButton" name="pushButtonStartServer">
      <property name="text">
       <string>Start Server</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QPushButton" name="pushButtonStopServer">
      <property name="text">
       <string>Stop Server</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="labelConnectionStatus">
      <property name="text">
       <string>Connection Status: unknown</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLineEdit" name="lineEditSaveFolder"/>
    </item>
    <item row="6" column="1">
     <widget class="QToolButton" name="toolButtonSaveFolder">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Save State Folder</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QToolButton" name="toolButtonLoadFolder">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="checkBoxCopySaves">
      <property name="text">
       <string>Copy all save states to separate folder</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayoutSearch">
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxKind">
        <item>
         <property name="text">
          <string>Enemy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Projectile</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Object</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NPC</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PlayerItem</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Manager</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditType">
        <property name="placeholderText">
         <string>Entity Type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSearchEntity">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="listViewFoundEntities"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
